airbyte-integrations/connectors/source-asana/source_asana/streams.py

```diff
@@ -141,6 +141,8 @@ def path(self, stream_slice: Mapping[str, Any] = None, **kwargs) -> str:
 
 
 class Projects(WorkspaceRequestParamsRelatedStream):
+    use_cache = True
```

Comment:
how does cache fix the issue? do we perform multiple identical requests within this stream?

Reply:
you can find more details in comment above